modules/cli-logging-in-web.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="cli-logging-in-web_{context}"]
+= Logging in to the OpenShift CLI using a web browser
+
+You can log in to the OpenShift CLI (`oc`) with the help of a web browser to access and manage your cluster. This allows users to avoid inserting their access token into the command line.
+
+== Logging in through a web browser with the CLI
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+* You must have a browser installed.
+
+.Procedure
+
+. Enter the `oc login` command with the '--web' flag:
++
+[source,terminal]
+----
+$ oc login <https://api.your-openshift-server.com> --web
+----
+
+[WARNING]
+====
+Logging in through the web browser runs a server on localhost with HTTP, not HTTPS. Use with caution on multi-user workstations.
+====
+
+. The web browser will open automatically. If it does not, click the link in the command output. If you do not specify the {product-title} server 'oc' will try to open the OpenShift Console of the cluster specified in the current 'oc' config file. If no 'oc' configuration exists, 'oc' prompts interactively for the server URL.
++
+.Example output
+
+[source,terminal]
+----
+$ oc login --web <1>
```

Comment:
Since you already listed the command in the above code block, this one should just be the example output. I'd remove this line, and move the <1> callout to the previous step's command.

Since you have an example command though you could always move it into a NOTE after the command block.

Though there is already a WARNING right after step 1, and we normally try to avoid back-to-back admonitions. This warning seems like something that doesn't have to be after step 1 though. I'd consider moving it up to before the prerequisites.